### PR TITLE
updates to help common issues. 

### PR DESCRIPTION
**Pull request by RaviTella**
Added the following:
1.	Common error 
2.	New images to show diagnostics folder. 
3.	Updated the Azure CLI instructions.

### CODE REVIEW

File: cosmos/sql/tools/java/ycsb/recipes/README.md [hunk 1]
```diff
@@ -58,25 +82,30 @@ A read recipe with a small read workload to familiarize you with the framework.
...
-      --resource-group <resource-group-name> \
-      --template-uri "https://raw.githubusercontent.com/Azure/azure-db-benchmarking/main/cosmos/sql/tools/java/ycsb/recipes/read/try-it-read/azuredeploy.json" \
-      --parameters parameter.json
-    ```
 7. re-executing the recipe by setting "Skip Load Phase" to "true" , while leaving the rest of the parameter values unchanged, will execute just the read phase of the workload again, using the VM from the previous execution. 
 
+## Common Errors
+Following are the most common user mistakes that lead to errors. The error logs will be available in a container within the storage account provided. The only exception being the first error listed below. A unreachable storage account. In which case, the longs will be available only in the VM.
+
+1. Following error will appear in "agent.out" in the "/home/benchmarking" of the client VM, if a incorrect storage connecting is passed.
```
Comment (simplynaveen20) [inline cosmos/sql/tools/java/ycsb/recipes/README.md:96]: Do these errors log in agent.err as well? It's easier to read err file in comparison to out

Comment (RaviTella) [reply cosmos/sql/tools/java/ycsb/recipes/README.md:96]: Application (YCSB) errors are not showing up in the error log, only script errors are showing up.

Comment (RaviTella) [reply cosmos/sql/tools/java/ycsb/recipes/README.md]: made the change for storage error log.

**Review by simplynaveen20 — COMMENTED** [review]
LGTM

**Review by simplynaveen20 — APPROVED** [review]

**Pull request merged**